source/common/grpc/common.h

```diff
@@ -25,6 +25,34 @@ class Exception : public EnvoyException {
 
 class Common {
 public:
+  enum GrpcStatus {
```

Comment:
gRPC status code are identical with protobuf status code, so just re-use it might be enough.